scripts/prepare_nightly_release.py

```diff
@@ -0,0 +1,123 @@
+"""Prepare a Rasa nightly release.
+
+- increases the version number to dev version provided
+"""
+import argparse
+import os
+import sys
+import toml
+from pathlib import Path
+from typing import Text
+from pep440_version_utils import Version, is_valid_version
+
+VERSION_FILE_PATH = "rasa/version.py"
+
+PYPROJECT_FILE_PATH = "pyproject.toml"
+
+
+def create_argument_parser() -> argparse.ArgumentParser:
+    """Parse all the command line arguments for the release script."""
+
+    parser = argparse.ArgumentParser(description="prepare the next nightly release")
+    parser.add_argument(
+        "--next_version", type=str, help="Rasa nightly version number",
+    )
+
+    return parser
+
+
+def project_root() -> Path:
+    """Root directory of the project."""
+    return Path(os.path.dirname(__file__)).parent
+
+
+def get_current_version() -> Text:
```

Comment:
Where is this function being used? I wasn't able to find where it's being called 🙈

Reply:
Very good catch, thanks, forgot to remove this after i removed the function that was calling it.